src/ResourceManager/DataMigration/Commands.DataMigration/Cmdlets/NewFileShare.cs

```diff
@@ -0,0 +1,44 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="NewFileShare.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+using System.Management.Automation;
+using Microsoft.Azure.Management.DataMigration.Models;
+
+namespace Microsoft.Azure.Commands.DataMigration.Cmdlets
+{
+    [Cmdlet(VerbsCommon.New, "AzureRmDataMigrationFileShare", SupportsShouldProcess = true), OutputType(typeof(MigrateSqlServerSqlDbDatabaseInput))]
```

Comment:
@hitenjava same comment about removing `SupportsShouldProcess`

Reply:
I had it as false earlier, but I got a build failure saying it should be true for 'New' type cmdlets.

Reply:
You can repress that warning for this cmdlet as well.